compiler/testData/diagnostics/foreignAnnotationsTests/tests/rxjava3Default.txt

```diff
@@ -0,0 +1,14 @@
+package
```

Comment:
Are rx* tests added to pr by mistake?

Reply:
Yes, looks like these files were autogenerated when I ran `:compiler:tests-common-new:test`. Someone should probably commit them.